decompressed/gui_file/www/lang/it-it/webui-mobiled.po

```diff
@@ -530,259 +530,259 @@ msgid "Unblock Retries Left:"
 msgstr "Tentativi rimanenti blocco:"
 
 msgid "IPv6"
-msgstr ""
+msgstr "IPv6"
 
 msgid "MNC:"
-msgstr ""
+msgstr "MNC:"
 
 msgid "Rx Packet Errors"
-msgstr ""
+msgstr "Errori pacchetti Rx"
 
 msgid "National"
-msgstr ""
+msgstr "Nazionale"
 
 msgid "Rx Packets"
-msgstr ""
+msgstr "Pacchetti RX"
 
 msgid "GB"
-msgstr ""
+msgstr "GB"
 
 msgid "Downlink Bandwidth:"
-msgstr ""
+msgstr "Banda download:"
 
 msgid "Total"
-msgstr ""
+msgstr "Totale"
 
 msgid "Tx Packet Errors"
-msgstr ""
+msgstr "Errori pachetti Tx"
 
 msgid "Path loss:"
-msgstr ""
+msgstr "Percorso perso:"
 
 msgid "GSM Only"
-msgstr ""
+msgstr "Solo GSM"
 
 msgid "RSCP:"
-msgstr ""
+msgstr "RSCP:"
 
 msgid "Active Band:"
-msgstr ""
+msgstr "Banda attiva:"
 
 msgid "LTE"
-msgstr ""
+msgstr "LTE"
 
 msgid "MSISDN"
-msgstr ""
+msgstr "MSISDN"
 
 msgid "Physical Cell ID:"
-msgstr ""
+msgstr "ID fisico cellulare:"
 
 msgid "RSSI:"
-msgstr ""
+msgstr "RSSI:"
 
 msgid "Data Usage Limit"
-msgstr ""
+msgstr "Limite uso dati"
 
 msgid "LTE Only"
-msgstr ""
+msgstr "Solo LTE"
 
 msgid "Uplink Bandwidth:"
-msgstr ""
+msgstr "Banda upload:"
 
 msgid "SMS"
-msgstr ""
+msgstr "SMS"
 
 msgid "Reset"
-msgstr ""
+msgstr "Ripristina"
 
 msgid "UMTS Preferred"
-msgstr ""
+msgstr "Preferenziale UMTS"
 
 msgid "Uplink EARFCN:"
-msgstr ""
+msgstr "EARFCN upload:"
 
 msgid "Uplink Frequency:"
-msgstr ""
+msgstr "Frequenza upload"
 
 msgid "CINR:"
-msgstr ""
+msgstr "CINR:"
 
 msgid "Monthly Reset Day"
-msgstr ""
+msgstr "Giorno ripristino mese"
 
 msgid "Tx Packets"
-msgstr ""
+msgstr "Pacchetti Tx"
 
 msgid "* Data usage is only an approximation. For actual usage, please refer to your network operator"
-msgstr ""
+msgstr "* L'uso dati è solo un'approssimazione.<br/>Per l'uso attuale chiedi al tuo operatore di rete."
 
 msgid "PIN"
-msgstr ""
+msgstr "PIN"
 
 msgid "IPv4"
-msgstr ""
+msgstr "IPv4"
 
 msgid "Auto"
-msgstr ""
+msgstr "Automatico"
 
 msgid "GSM"
-msgstr ""
+msgstr "GSM"
 
 msgid "Reset Data Usage"
-msgstr ""
+msgstr "Azzera uso dati"
 
 msgid "Uplink ARFCN:"
-msgstr ""
+msgstr "ARFCN upload:"
 
 msgid "Radio Bearer:"
-msgstr ""
+msgstr "Bearer radio:"
 
 msgid "EARFCN"
-msgstr ""
+msgstr "EARFCN"
 
 msgid "Downlink UARFCN:"
-msgstr ""
+msgstr "UARFCN download:"
 
 msgid "RSRP:"
-msgstr ""
+msgstr "RSRP:"
 
 msgid "Total Volume"
-msgstr ""
+msgstr "Violume totale"
 
 msgid "Data Usage"
-msgstr ""
+msgstr "Uso dati"
 
 msgid "MB"
-msgstr ""
+msgstr "MB"
 
 msgid "International"
-msgstr ""
+msgstr "Internazionale"
 
 msgid "Device Status And Settings"
-msgstr ""
+msgstr "Stato dispositivo ed impostazioni"
 
 msgid "ECIO:"
-msgstr ""
+msgstr "ECIO:"
 
 msgid "Downlink ARFCN:"
-msgstr ""
+msgstr "ARFCN download:"
```

Comment:
non credo sia necessario tradurre

Reply:
Perchè lasciarlo in inglese?
La definizione in Italiano c'è.

Reply:
il problema sta nel cambiare downlink con download